### DISCUSSION TITLE
10.10.0 EDL Configuration

### DISCUSSION THREAD

**Discussion by seang96** [intro-skipper/intro-skipper#361] [Q&A] [answered]
I have a couple questions on how to properly set it up with media segments.

1. How does the "Update media segments for newly added first during scan" work? Should EDL creator "Update EDL files" be unchecked with this?
2. Does the checkbox for newly added checkbox handle the order of scheduled tasks for media segments for new items? If not what is the proper order? Detect intro and credits, create EDL, then media segment scan?

Thanks for keeping this plugin updated and well maintained!

**Comment by jumoog** [comment]
Have you installed the latest version? EDL support has been removed.

**Reply by seang96** [reply]
Yes, that is why I assume the EDL Creator would be required or is it not required because intro skipper skips edl creation and works with media segments directly?

**Comment by rlauuzo** [comment]
The "Update media segments for newly added first during scan" option automatically updates media segments in the Jellyfin database during the scan, effectively handling the "Scan Media Segments" task for you.

If you disable this option, you'll need to follow a manual task order to ensure accurate updates:

1. Run the "Intro Skipper" task to detect intros and credits.

2. After it completes, run the "Scan Media Segments" task to update the media segments in the database.

**Reply by seang96** [reply]
That answers one of my questions, thanks!

What about EDL files, do they still need to be processed with the new EDL Creator plugin? What is the benefit of making them if they don't need to be processed?

**Reply by rlauuzo** [reply]
EDL files are generated from media segments, and write permissions are required for your library. Just ensure the Intro Skipper task runs first (and run Scan Media Segments as well if the option is disabled).

**Reply by jumoog** [reply, marked as answer]
You can use the EDL files for Kodi. Jellyfin don't uses them.
